packages/instrumentation/src/ActiveTracingHelper.ts

```diff
@@ -21,12 +23,15 @@ const nonSampledTraceParent = `00-10-10-00`
 
 type Options = {
   traceMiddleware: boolean
+  ignoreLayersTypes: PrismaLayerType[]
```

Comment:
Could you clarify why you use `layer` name for this feature?
We are filtering out spans, OTEL docs also call them spans. What would you think about renaming this property to `ignoreSpanNames`?

Reply:
No issues with that. The `layer` name was meant to follow Prisma naming as it was the target. Happy to update it